tests/ert_tests/ert3/evaluator/test_evaluator.py

```diff
@@ -99,68 +101,6 @@ def test_evaluator_script(
     assert expected == evaluation_records
 
 
-@pytest.mark.requires_ert_storage
-@pytest.mark.parametrize("coeffs, expected", TEST_PARAMETRIZATION)
-def test_evaluator_function(
-    workspace, function_stages_config, base_ensemble_dict, coeffs, expected
-):
-    storage_path = workspace._path / ".ert" / "tmp" / "test"
-    input_transmitters = get_inputs(coeffs)
-    base_ensemble_dict["size"] = len(coeffs)
-    base_ensemble_dict["storage_type"] = "shared_disk"
-    ensemble_config = ert3.config.load_ensemble_config(base_ensemble_dict)
-
-    experiment_run_config = ert3.config.ExperimentRunConfig(
-        ert3.config.ExperimentConfig(type="evaluation"),
-        function_stages_config,
-        ensemble_config,
-        ert3.config.ParametersConfig.parse_obj([]),
-    )
-    stage = experiment_run_config.get_stage()
-
-    step_builder = (
-        create_step_builder()
-        .set_name(f"{stage.name}-only_step")
-        .set_type("function" if isinstance(stage, ert3.config.Function) else "unix")
-    )
-
-    inputs = experiment_run_config.get_linked_inputs()
-    stochastic_inputs = tuple(inputs[ert3.config.SourceNS.stochastic].values())
-
-    ert3.evaluator.add_step_inputs(stochastic_inputs, input_transmitters, step_builder)
-
-    ert3.evaluator.add_step_outputs(
-        ensemble_config.storage_type,
-        stage,
-        storage_path,
-        ensemble_config.size,
-        step_builder,
-    )
-
-    if isinstance(stage, ert3.config.Unix):
-        ert3.evaluator.add_commands(
-            stage.transportable_commands,
-            base_ensemble_dict["storage_type"],
-            storage_path,
-            step_builder,
-        )
```

Comment:
Not quite sure what this was supposed to do yet, but it was never `Unix`, so this was never executed.

Reply:
Probably a copy paste error, as the same code snippet _is_ called in `test_evaluator_script`